sql/moz-fx-data-shared-prod/apple_ads_external/ios_app_campaign_stats_v1/bigconfig.yml

```diff
@@ -21,6 +23,8 @@ tag_deployments:
 
   - collection:
       name: Operational Checks
+      notification_channels:
+        - slack: '#ds-bigeye-triage'
```

Comment:
Can we please replace the slack channel to #de-bigeye-triage? All the operation checks i.e. volume and freshness would be directed to this collection that DE will be responsible for triaging.

Reply:
ah sure, sorry. I thought there was only one slack channel right now.